sdk/ai/ai-projects/test/public/files/files.spec.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import type { Recorder, VitestTestContext } from "@azure-tools/test-recorder";
+import { createRecorder, createProjectsClient } from "../utils/createClient.js";
+import { assert, beforeEach, afterEach, it, describe } from "vitest";
+import type { AIProjectClient } from "../../../src/index.js";
+import type { OpenAI } from "openai/client";
+
+const testMode = (process.env.TEST_MODE ?? "playback").toLowerCase();
```

Comment:
The test recorder exports helper that tells you whether in playback mode, e.g. isPlaybackMode()